src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
     />
```

Comment:
SCSS files should not be linked directly in HTML. Instead, compile them to CSS and link the CSS file. SCSS is a preprocessor and won't be understood by the browser.